### DISCUSSION TITLE
Development help

### DISCUSSION THREAD

**Discussion by tsouza** [DeFiCh/app#1176] [Q&A]
Hello DeFiCh engineers,

I am a software engineer researching blockchain related code. Currently, I'm trying to understand how wrapped tokens, such as USDC and BTC, works in DeFiChain. I tried searching the github repo but could not identify anything. Could someone please point me out some code references in the github repo or any other resource around this subject?

Best

**Comment by TatyOko28** [comment]
Hi,

Wrapped tokens like USDC and BTC on DeFiChain operate as DFI-backed tokens that mirror their real-world counterparts, typically through an over-collateralized mechanism or tokenization process. You can find relevant code references in the [DeFiChain repository](https://github.com/DeFiCh/ain) under src/masternodes and src/tx for transaction logic related to token wrapping and minting. Additionally, the DFI Token Standard and DFI-Bridge components might be relevant, and you can explore DeFiChain's Token Documentation for further insights. Let us know if you need more specific guidance!

Best
